uaclient/contract.py

```diff
@@ -187,16 +187,19 @@ def report_machine_activity(self, enabled_services: "List[str]"):
         # provided by the server. We expect the response to be
         # a partial representation of the machine token json
         if response:
+            # the activitu information received as a response here
+            # will not provide the information inside an activityInfo
+            # structure. However, this structure will be reflected when
+            # we reach the contract for attach and refresh requests.
+            # Because of that, we will manually create this structure here.
+            response = {"activityInfo": response}
+
             machine_token = self.cfg.read_cache("machine-token")
             depth_first_merge_overlay_dict(
                 base_dict=machine_token, overlay_dict=response
             )
             self.cfg.write_cache("machine-token", machine_token)
 
-            machine_id = response.get("machineTokenInfo", {}).get("machineId")
-            if machine_id:
-                self.cfg.write_cache("machine-id", machine_id)
```

Comment:
Were we using this anywhere else? Do we still need the "machine-id" cache item?

Reply:
Originally, the machine activity endpoint would return a new `machine-id` if needed. However this was replaced by the `activityID` token. That's why I dropped it here

Reply:
Got it that makes sense! So can we drop line 98 of `uaclient/config.py` in this PR as well?

Reply:
Yep, good catch